.github/workflows/dependencies/run-dotnet-whatsnew.sh

```diff
@@ -20,4 +20,4 @@ done
 dotnet whatsnew \
     --owner $OWNER \
     --repo $REPO \
-    --savedir $SAVEDIR
+    --savedir $SAVEDIR
```

Comment:
@LayersOfAbstraction ... Add the return back to this line ...

```suggestion
    --savedir $SAVEDIR

```